### ISSUE TITLE
As an ECAD dev rel, I want the scaffold task to be fixed and all scaffolds updated so that documentation and the end user experience remains consistent when 0.4.0 is released

### ISSUE THREAD

**Issue by mweichert** [ecadlabs/taqueria#736]
The following adjustments need to be made once the 620-operations branch is merged into main:
- [x] In cli.ts, adjust the `scaffoldProject()` function to not checkout a branch after git clone @mweichert 
- [x] In e2e/taqueria-cli-scaffolding-functional.spec.ts, reactivate all other tests @alexzbusko 

## Issue Number 1: @mweichert 

The main branch of the taqueria-scaffold-quickstart repo has a .taq/config.json file which is using an outdated schema, and therefore doesn't work with the 620-operations branch.

To workaround this, I've added a 620-operations branch to the taqueria-scaffold-quickstart repo which contains an updated schema. I then adjusted the `scaffoldProject()` function to not checkout this branch after cloning.

## Issue Number 2 @alexzbusko 

The taqueria-scaffold-quickstart repo has the latest official version of taqueria plugins installed from the NPM registry. These plugin returns a pluginInfo schema which has also been outdated. As such, I disabled some of the tests in e2e/taqueria-cli-scaffolding-functional.spec.ts

## Issue Number 3: @hu3man 

Release checklist:
- [x] Run `npm i` to update package.json files for each PR
- [x] Merge scaffold PR's
- [x] Update Documentation

Make PR's to Update Scaffolds:
- [x] taqueria-scaffold-quickstart-demo (https://github.com/ecadlabs/taqueria-scaffold-quickstart/pull/2)
- [x] taqueria-scaffold-demo (https://github.com/ecadlabs/taqueria-scaffold-demo/pull/3)
- [x] taqueria-scaffold-hello-tacos-tutorial (unnecessary)
- [x] taqueria-scaffold-taco-shop (https://github.com/ecadlabs/taqueria-scaffold-taco-shop/pull/1)

**Comment by mweichert** [comment]
@alexzbusko adding you here so that you're aware, and so that you can possibly brainstorm how to handle this going forward.

**Comment by hu3man** [comment]
This will need to include the new scaffolds as well (hello-tacos, taco-shop, demo) and I can take on updating all 4

Just to confirm @mweichert , the changes to the scaffold task are just in your branch to get the tests passing right? 

If so, then would this ticket would be implemented immediately after merge (timed with updates to the scaffolds)?

**Comment by mweichert** [comment]
@hu3man I'll need to create a branch for those other scaffolds as well.
